README.md

```diff
@@ -5,14 +5,26 @@
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2FTykTechnologies%2Ftyk.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2FTykTechnologies%2Ftyk?ref=badge_shield)
 --
 
-Tyk is an open source Enterprise API Gateway. This README will describe the API gateway itself. To get to our commercial offering please use the following links
+This README describes how to get started with the Tyk open source API gateway.
 
-[<img src="https://raw.githubusercontent.com/TykTechnologies/tyk-docs/master/tyk-docs/assets/img/logos/tyk-logo-cloud.png" width="9%" >](https://tyk.io/docs/tyk-cloud/)
+Tyk is a complete API lifecycle management solution; our open source API Gateway is secure, lightweight, [highly performant](https://tyk.io/performance-benchmarks/) and is an integral component of our [open source](https://tyk.io/docs/tyk-stack/#tyk-open-source) and [licensed](https://tyk.io/docs/tyk-stack/#closed-source) options.
```

Comment:
@chrisanderton can you please check the info you have added isn't being repeated under the section `Tyk Open Source API Gateway`

Reply:
How about this shorter version..